README.md

```diff
@@ -96,6 +96,7 @@ Many variables have default values and are therefore optional.
 | `opening-delimiter`           | `` ```javascript ``   | The opening delimiter of doctest blocks in the source files.                  |
 | `closing-delimiter`           | `` ``` ``             | The closing delimiter of doctest blocks in the source files.                  |
 | `version-tag-prefix`          | `v`                   | The prefix of annotated version tags (`version-tag-prefix =` for no prefix).  |
+| `module-type`                 | `commonjs`            | The type of module doctest should use. One of `commonjs`, `amd`, or `esm`.    |
```

Comment:
Let's include `module-type` after `closing-delimiter` so the doctest-related variables are together.

Reply:
:zap: